packages/js/src/plugins/tokenModule/TokenClient.ts

```diff
@@ -45,10 +45,31 @@ import {
   sendTokensOperation,
   SendTokensOutput,
 } from './sendTokens';
+import {
+  AddMetadataInput,
+  addMetadataOperation,
+  AddMetadataOutput,
+} from './addMetadata';
+import { MintWithMetadata } from '@/plugins';
 
 export class TokenClient {
   constructor(protected readonly metaplex: Metaplex) {}
 
+  addMetadata(
```

Comment:
As discussed on Slack, would you mind moving that operation to the NFT module instead?